tests/pytests/test_json.py

```diff
@@ -1117,3 +1117,31 @@ def testRedisCommands(env):
         time.sleep(0.1)
         env.expect('JSON.GET', 'doc:1', '$').equal(None)
         env.expect('ft.search', 'idx', 'ri*', 'NOCONTENT').equal([0])
+
+@no_msan
+def testUpperLower(env):
+
+    env = Env(moduleArgs='DEFAULT_DIALECT 3')
+    conn = getConnectionByEnv(env)
+
+    # create index
+    env.assertOk(conn.execute_command('FT.CREATE groupIdx  ON JSON PREFIX 1 group: SCHEMA $.tags.* AS tags TAG'))
```

Comment:
Need the command name to be separated, so it can be used to determine the key position (defined [here](https://github.com/RediSearch/RediSearch/blob/e318a207a596678e878da0513184136b558b4ff9/coord/src/rmr/command.c#L82))
```suggestion
    env.assertOk(conn.execute_command('FT.CREATE' ,'groupIdx' ,'ON', 'JSON', 'PREFIX', 1, 'group:', 'SCHEMA', '$.tags.*', 'AS', 'tags', 'TAG'))
```